File: _includes/code/fibonacci.py
#!/usr/bin/env python
import sys

def fibonacci(i):
    if i <= 1:
        return i
    else:
        return fibonacci(i-1) + fibonacci(i-2)

if len(sys.argv) < 2:
    print "Usage: %s number" % sys.argv[0]
else:
    n = int(sys.argv[1])
    print "F(%d) = %d" % (n, fibonacci(n))

File: _includes/code/fibonacci.rb
#!/usr/bin/ruby

def fibonacci(i)
	if i <= 1
		return i
	else
		return fibonacci(i-1) + fibonacci(i-2)
	end
end

if ARGV.length < 1
	puts "Usage: #{$0} number"
else
	n = ARGV[0].to_i
	puts "F(#{n}) = #{fibonacci(n)}"
end
